packages/vscode-extension/src/commands/CreateProject.ts

```diff
@@ -0,0 +1,229 @@
+            defaultUri: Uri.file(os.homedir()),
+        });
+        if (!selectedFolder) {
+            return {};
+        }
+
+        let projectType = await this.showSelectProjectTypeQuestion();
+        if (!projectType) {
+            return {};
+        }
+        commandArgs[COMMAND_ARGUMENTS.TYPE] = this.translationService.getMessage(CREATE_PROJECT.PROJECT_TYPE.SUITEAPP)
+            ? ApplicationConstants.PROJECT_ACP
+            : ApplicationConstants.PROJECT_SUITEAPP;
+
+        const projectName = await this.showProjectNameQuestion();
+        if (!projectName) {
+            return {};
+        }
+        commandArgs[COMMAND_ARGUMENTS.PROJECT_NAME] = projectName;
+
+        let publisherId, projectId, projectVersion;
+        if (projectType === this.translationService.getMessage(CREATE_PROJECT.PROJECT_TYPE.SUITEAPP)) {
+            publisherId = await this.showPublisherIdQuestion();
+            if (publisherId === undefined) {
+                return {};
+            }
+            commandArgs[COMMAND_ARGUMENTS.PUBLISHER_ID] = publisherId;
+
+            projectId = await this.showProjectIdQuestion();
+            if (!projectId) {
+                return {};
+            }
+            commandArgs[COMMAND_ARGUMENTS.PROJECT_ID] = projectId;
+
+            projectVersion = await this.showProjectVersionQuestion();
+            if (!projectVersion) {
+                return {};
+            }
+            commandArgs[COMMAND_ARGUMENTS.PROJECT_VERSION] = projectVersion;
+
+            commandArgs[COMMAND_ARGUMENTS.PROJECT_FOLDER_NAME] = publisherId + '.' + projectId;
+        } else {
+            commandArgs[COMMAND_ARGUMENTS.PROJECT_FOLDER_NAME] = projectName;
+        }
+        commandArgs[COMMAND_ARGUMENTS.PARENT_DIRECTORY] = path.join(selectedFolder[0].fsPath, commandArgs[COMMAND_ARGUMENTS.PROJECT_FOLDER_NAME]);
+
+        const includeUnitTesting = await this.showIncludeUnitTestingQuestion();
+        if (!includeUnitTesting) {
+            return {};
+        }
+        commandArgs[COMMAND_ARGUMENTS.INCLUDE_UNIT_TESTING] = includeUnitTesting === this.translationService.getMessage(ANSWERS.NO) ? 'false' : 'true';
+
+        const projectAbsolutePath = commandArgs[COMMAND_ARGUMENTS.PARENT_DIRECTORY];
+        const fileSystemService = new FileSystemService();
+        if (fileSystemService.folderExists(projectAbsolutePath) && !fileSystemService.isFolderEmpty(projectAbsolutePath)) {
+            const overwriteProject = await this.showOverwriteFilesQuestion(projectAbsolutePath);
+            if (!overwriteProject) {
+                return {};
+            }
+            commandArgs[COMMAND_ARGUMENTS.OVERWRITE] = overwriteProject === this.translationService.getMessage(ANSWERS.NO) ? 'false' : 'true'; 
+        }
+
+        return commandArgs;
+    }
+
+    private showOverwriteFilesQuestion(projectAbsolutePath: string): Thenable<string | undefined> {
```

Comment:
In this method you are not only showing the questions but actually getting the result (more exactly the promise, that when using "awaits")
I would name it getOverwriteFiles directly or if you prefer to mantain the "Question" suffix then I would go for "promptOverwriteFilesQuestion" that have the extraction of the response implicit

Reply:
I updated it and changed itto prompt... as you suggested.